### ISSUE TITLE
Bedrock Armorstand Placement Issue

### ISSUE THREAD

**Issue by bubbakadriu** [GeyserMC/Geyser#3676]
### Describe the bug

When java attached an armorstand to a player, it attaches it to the correct position. On bedrock however the armorstand is way above the player. Plugins using backpack cosmetics use armorstands attached to the player to not take up inventory slot. As seen in the pictures the armorstand is not attaching correctly on bedrock

### To Reproduce

Plugins this has occured with are MCCosmetics, HMCCosmetics & CMI (/ride)

Setup a backpack and ensure that it is a model that is visible (custom resource pack item)

### Expected behaviour

The armorstand should be inside the bedrock player in the same position as it is on java

### Server Version and Plugins

git-purpur-1845 1.19.2
Geyser
MCCosmetics
CMI
HMCCosmetics

### Geyser Dump

https://dump.geysermc.org/X8nN7QwX9LtXvoCQ2bsKydukmBOaZ2BV

### Geyser Version

2.1.0-SNAPSHOT (git-master-03390b9)

### Minecraft: Bedrock Edition Device/Version

Apple IPhone (all bedrock devices)

**Comment by bubbakadriu** [comment]
![image](https://user-images.githubusercontent.com/38432044/231754967-b52ee7bb-439a-43bc-bd1b-43335dbe6962.png)

**Comment by bubbakadriu** [comment]
![image](https://user-images.githubusercontent.com/38432044/231755115-ad4bbb73-2c60-4d49-9fcd-89b4437b998f.jpeg)

**Comment by bubbakadriu** [comment]
bump

**Comment by onebeastchris** [comment]
Should be resolved by https://github.com/GeyserMC/Geyser/pull/6595

**Comment by Nogard-YT** [comment]
> Should be resolved by [#6595](https://github.com/GeyserMC/Geyser/pull/6595)

Yes, and it's also worth mentioning that the attachable needs to be converted properly, with the right display property transformations.
